cmd/up/up.go

```diff
@@ -236,14 +245,14 @@ okteto up my-svc -- echo this is a test
 				return fmt.Errorf("failed to load k8s client: %w", err)
 			}
 
-			if upOptions.Deploy || !pipeline.IsDeployed(ctx, up.Manifest.Name, up.Manifest.Namespace, k8sClient) {
+			if upOptions.Deploy || !pipeline.IsDeployed(ctx, up.Manifest.Name, okteto.GetContext().Namespace, k8sClient) {
```

Comment:
As we already have it in the upContext. Should we start using `up.Namespace` instead?